botorch/acquisition/max_value_entropy_search.py

```diff
@@ -452,7 +721,7 @@ def forward(self, X: Tensor) -> Tensor:
             X=X_expand, mean_M=mean_M, variance_M=variance_M, covar_mM=covar_mM
         )
         ig = self.cost_aware_utility(X=X, deltas=ig, sampler=self.cost_sampler)
-        return ig.mean(dim=0)  # average over the fantasies
+        return ig  # average over the fantasies
```

Comment:
This change makes the tests of MFMES fail, since the self.cost_aware_utility() requires an input of #num_fantasies x (batch_shape).